README.md

```diff
@@ -29,14 +29,15 @@ C_SOURCES := $(filter-out Middlewares/Third_Party/FreeRTOS/Source/portable/MemMa
 # Add micro-ROS utils
 C_SOURCES += microros_component/custom_memory_manager.c
 C_SOURCES += microros_component/microros_allocators.c
-C_SOURCES += microros_component/microros_transports.c
 C_SOURCES += microros_component/microros_time.c
+C_SOURCES += microros_component/microros_transports/dma_transport.c
+#C_SOURCES += microros_component/microros_transports/it_transport.c
```

Comment:
In my opinion, we should provide here only one line and explain in a comment that the transport implementation should be added here.